src/Tezos/Forging.cpp

```diff
@@ -17,12 +18,57 @@ using namespace TW;
 using namespace TW::Tezos;
 using namespace TW::Tezos::Proto;
 
+namespace {
+
+constexpr const char* gTezosContractAddressPrefix{"KT1"};
+
+void prefixEncoder(const std::string& address, Data& forged) {
```

Comment:
`Encoder` is not an action (for a method)
```suggestion
void encodePrefix(const std::string& address, Data& forged) {
```